src/main/java/crazypants/enderzoo/entity/EntityEnderminy.java

```diff
@@ -123,7 +123,7 @@ public boolean getCanSpawnHere() {
    */
   private boolean shouldAttackPlayer(EntityPlayer player) {
 
-    ItemStack itemstack = player.inventory.armorInventory[3];
+    ItemStack itemstack = player.inventory.armorInventory.get(3);
```

Comment:
can you check if slot 3 is the correct one? I don't know if this was updated correctly after 1.7. Thx.